scapy/layers/inet6.py

```diff
@@ -3995,3 +4024,4 @@ def _load_dict(d):
 bind_layers(IPv6, IPv6, nh=socket.IPPROTO_IPV6)
 bind_layers(IPv6, IP, nh=socket.IPPROTO_IPIP)
 bind_layers(IPv6, GRE, nh=socket.IPPROTO_GRE)
+bind_layers(IPv6, RPL, nh=58, dst="ff02::1a")
```

Comment:
`ICMPv6MLQuery()` seems to be using `overload_fields` for overriding the dst-address and next-header. Actually I used the same initially but was suggested to use `bind_layers` by @gpotter2

Reply:
I meant `bind_top_down`.

IPv6 uses a lot of  `overload_fields` (which could be replaced by `bind_top_down` calls). It has the exact same effect though i prefer the second option because it "looks" fancier. But in this case you might want to remain consistent as guedou said.

Reply:
Thanks @gpotter2

Reply:
Hi @gpotter2 , Can you please review the change in rpl.py again? I could not get the dissection to work without `guess_payload_class`.

Dissection works perfectly fine with `guess_payload_class`. I realized this (that dissection does not work with bind_layer but works with guess_payload_class) after adding more Unit tests.

Reply:
It's fine :+1: